src/traces/cone/convert.js

```diff
@@ -48,25 +49,44 @@ function convert(scene, trace) {
 
 proto.update = function(data) {
     this.data = data;
-    this.mesh.update(convert(this.scene, data));
+
+    var meshData = convert(this.scene, data);
+
+    this.mesh.update(meshData);
+    this.pts.update({position: meshData._pts});
 };
 
 proto.dispose = function() {
+    this.scene.glplot.remove(this.pts);
+    this.pts.dispose();
     this.scene.glplot.remove(this.mesh);
     this.mesh.dispose();
 };
 
-function createMesh3DTrace(scene, data) {
+function createConeTrace(scene, data) {
     var gl = scene.glplot.gl;
-    var meshData = convert(scene, data);
-    var mesh = createMesh(gl, meshData);
-    var result = new Mesh3DTrace(scene, mesh, data.uid);
 
-    result.data = data;
-    mesh._trace = result;
+    var meshData = convert(scene, data);
+    var mesh = createConeMesh(gl, meshData);
+
+    var pts = createScatterPlot({
```

Comment:
a little bit hacky and not very efficient, but this works:

![peek 2018-05-16 17-24](https://user-images.githubusercontent.com/6675409/40145345-6e0f0b8c-592f-11e8-89a2-321554c42429.gif)

Reply:
Nice, that was fast! Is it possible to get `(u, v, w)` in the hovertext as well?

Reply:
It's possible!

![peek 2018-05-16 23-31](https://user-images.githubusercontent.com/6675409/40155378-639c1cda-5961-11e8-86d0-5c2e3184295a.gif)

but maybe someone would argue that 6 fields in one hover label in too much by default. Maybe we could just show the vector norm value? Or maybe have `hoverinfo: 'x+y+z+norm'` by default and optionally up to: `x+y+z+u+v+w+norm+text`

Reply:
> `'x+y+z+norm'` by default and optionally up to: `x+y+z+u+v+w+norm+text`

I like it!

Reply:
done in https://github.com/plotly/plotly.js/pull/2641/commits/e4a203563ff02dca646aa904eda423da6df601d4